app/views/user_mailer/swap_cancelled.haml

```diff
@@ -1,9 +1,10 @@
 %p
 	Hi #{@user.name},
 %p
-	I'm sorry to say that your swap with #{@swap_with.name} has been cancelled.
-	This could have happened if you or #{@swap_with.name} changed preferred parties, moved constituency,
-	declined the swap, or (most likely!) if the swap expired before being confirmed.
+	I'm sorry to say that your swap with #{@swap_with&.name} has been cancelled.
```

Comment:
Again this will interpolate `nil` into the text which we don't want - I'll make the mention of the name conditional.